### PR TITLE
Fixing the testing samples to conform with DFD3

### PR DESCRIPTION
**Pull request by izar**
Errr....i forgot to fix the tests when changing to DFD3. All the html on the saved compares is not needed anymore.

### CODE REVIEW

File: tests/dfd.dot [hunk 1]
```diff
@@ -62,22 +59,19 @@ digraph tm {
...
-                    </table>
-                >;
+                label = "Web Server";
+                margin = 0.02;
             ]
 
             datastore_SQLDatabase_0291419f72 [
                 shape = none;
+                fixedsize = shape;
+                image = "/Users/itarandach/Src/pytm/pytm/images/datastore.png";
```
Comment (nineinchnick) [inline tests/dfd.dot]: This will make it fail for anyone else. Can we make it a relative path?

**Review by nineinchnick — APPROVED** [review]

File: tests/dfd.dot (hunk 1 above)
Comment (izar) [reply tests/dfd.dot]: Just figuring that out now, the hard way...

Comment (izar) [reply tests/dfd.dot]: suggestions? the path appears in 3 files that we compare output to, and it is hardcoded there.

Comment (nineinchnick) [reply tests/dfd.dot]: It's being set to an absolute path on line 1406 in pytm.py. This is probably required, especially when pytm will be installed using pip. So we'd need to fix it in the test, before comparing files.

Comment (izar) [reply tests/dfd.dot]: i'll do it when the file is read.

Comment (izar) [reply tests/dfd.dot]: holly molly i think it works now.

**Pull request merged**